packages/server/__tests__/api/grants.test.js

```diff
@@ -729,8 +729,16 @@ HHS-2021-IHS-TPI-0001,Community Health Aide Program:  Tribal Planning &`;
         });
     });
     context('GET /exportCSVRecentActivities', () => {
-        it('produces the expected column headers', async () => {
-            const expectedCsvHeaders = 'Date,Team,Grant,Status Code,Grant Assigned By,Email';
+        let originalFeatureFlagValue;
+        before(() => {
+            originalFeatureFlagValue = process.env.SHARE_TERMINOLOGY_ENABLED;
+        });
+        after(() => {
+            process.env.SHARE_TERMINOLOGY_ENABLED = originalFeatureFlagValue;
+        });
+
+        it('produces the expected column headers', async () => { 
+            const expectedCsvHeaders = process.env.SHARE_TERMINOLOGY_ENABLED ? 'Date,Team,Grant,Status Code,Grant Shared By,Email' : 'Date,Team,Grant,Status Code,Grant Assigned By,Email';
```

Comment:
Thanks for updating the test! Couple quick tweaks: 

1. I think this is better expressed as two separate tests (one where the flag is one, one where it's off). Otherwise, you end up duplicating the logic you're testing inside the test itself, which isn't as helpful. 
2. Looks like linting issues cropped up here again. If you're having any trouble getting autofix on save working in your editor, I'd love to help and make sure the steps get properly documented!

Reply:
After some updates/more set-up, I think linting is working now for me. Thank you for offering to help!
I split the test into two, but I'm not sure if the code after the flag and csv assignments is a bit repetitive now?

Reply:
Ah, good call! I don't think the repetitiveness is bad enough to block this PR if you want to land it, but I agree it's probably cleaner as three separate tests, something like this: 

```js
context('GET /exportCSVRecentActivities', () => {
  let originalShareTerminologyEnabled;

  before(() => {
    originalShareTerminologyEnabled = process.env.SHARE_TERMINOLOGY_ENABLED;
  });

  after(() => {
    process.env.SHARE_TERMINOLOGY_ENABLED = originalShareTerminologyEnabled;
  });

  it('returns valid CSV response', async () => {
    const response = await fetchApi('/grants/exportCSVRecentActivities', agencies.own, fetchOptions.staff);
    expect(response.statusText).to.equal('OK');
    expect(response.headers.get('Content-Type')).to.include('text/csv');
    expect(response.headers.get('Content-Disposition')).to.include('attachment');
  });

  it('includes correct column headers with share terminology disabled', async () => {
    process.env.SHARE_TERMINOLOGY_ENABLED = 'false';
    const response = await fetchApi('/grants/exportCSVRecentActivities', agencies.own, fetchOptions.staff);
    expect(await response.text()).to.contain('Date,Team,Grant,Status Code,Grant Assigned By,Email');
  });

  it('includes correct column headers with share terminology enabled', async () => {
    process.env.SHARE_TERMINOLOGY_ENABLED = 'true';
    const response = await fetchApi('/grants/exportCSVRecentActivities', agencies.own, fetchOptions.staff);
    expect(await response.text()).to.contain('Date,Team,Grant,Status Code,Grant Shared By,Email');
  });
});
```